redhat-cop/ah_configuration
# Add collection manage plugin

### What does this PR do?
Add a collection manage plugin
        - If present will return data on a collection.
        - If present with version, will return data on a collection version.
        - If absent without version, will delete the collection and all versions.
        - If absent with version, will delete only specified version.

### How should this be tested?
Automated tests

I wonder if I should merge in the functionality of ah_collection_upload to this, Upload if specified path, then get its data, and delete if state is absent.
